.github/workflows/ruby.yml

```diff
@@ -27,10 +27,10 @@ jobs:
       run: |
         apt-get update
         apt-get install -y libdb-dev
-    - name: Install bundler
+    - name: Update rubygems & bundler
       run: |
         ruby -v
-        gem install bundler
+        gem update --system
```

Comment:
[note] The test for `Gem.plugindir` fails with Ruby 2.7 because Ruby 2.7 includes RubyGems 3.1 by default.

> Error: test_plugindir(GemSingletonTest): NoMethodError: undefined method `plugindir' for Gem:Module

https://github.com/ruby/rbs/pull/605/checks?check_run_id=1937717810#step:8:1248

`.plugindir` has been available since RubyGems 3.2 (see <https://github.com/rubygems/rubygems/pull/3163>)

So, I've updated this CI configuration to use the latest RubyGems.
This way may be correct, but it seems to me that this CI update should be done on another PR.

Can you provide me feedback, please?

Reply:
Installing the latest rubygems looks reasonable. 👍